src/main/antlr4/imports/mysql_alter_table.g4

```diff
@@ -62,6 +62,7 @@ ignored_alter_specifications:
      */
      | ALGORITHM '='? algorithm_type
      | LOCK '='? lock_type
+     | RENAME (INDEX|KEY) name TO name
```

Comment:
wondering if it'd be useful to indicate 5.5/5.6/5.7 specific syntax, e.g. dropping 5.5 support